include/meta/rtti/EnableRTTI.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "StaticTypedesc.hpp"
+#include <concepts>
+
+#include <meta/ThisT.h>
+
+/// @brief Unique RTTI type name macro for internal usage
+#define _ARCH_RTTI_UNIQUE _RTTIUnique
+
+/// @brief Class attribute enabling typedesc() macro usage (requires THIS_T attribute to be set)
+/// @brief Obsolete for non-polymorphic types
+/// @see std::is_polymorphic_v
+/// @details If not used, typedesc() will return incorrect TypeDescriptor when used on base class pointer or reference
+#define EnableRTTI                                                                                                  \
```

Comment:
I think all of our macros should have a prefix (eg. `Arch_`) as a namespace, so user wouldn't have a problem wiht name collisions